client/src/api/rpaFramework.js

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
I thaught about that but we are not just selecting applications in that file. Thats why I think it could be misleading. 
What do you think about the name "rpaFrameworkCalls"? @kej-jay

Reply:
I get your point, but I'm still not a huge fan of including the _rpaFramework_ in the name

Reply:
I think it is a good idea to keep rpaFramework in the name because all the api calls in that file are directed at the rap-framework route in the backend. Also this file is in the api folder so it should match the backend structure :)

Reply:
Ideally we would not have the need for RPA Framework, what we do with those calls is retrieve available applications and tasks, which for now in our case happens to be rpa framework. If we decide to change that later on, it should not require to rename all files.
I should have been more careful about this in the backend as well

Reply:
I called the file now "applicationAndTaskSelection"